src/contentscript/injector.ts

```diff
@@ -484,26 +484,18 @@ export class Injector {
       }
 
       // ToDo: generalize loading of parser configs
-      if (manifest.name === 'twitter-adapter.dapplet-base.eth') {
+      if (manifest.type === ModuleTypes.ParserConfig && typeof scriptOrConfig === 'object') {
```

Comment:
It's better to move checking of `typeof scriptOrConfig === 'object'` inside. Otherwise it will be difficult to understand the error.

```typescript
if (manifest.type === ModuleTypes.ParserConfig) {
   if (typeof scriptOrConfig !== 'object') {
      throw new Error("shit happens");
   }
   // ...
}
```